docs/static/resources/coraza-deployment.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: coraza-spoa
+  name: coraza-spoa
+  namespace: ingress-controller
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      run: coraza-spoa
+  template:
+    metadata:
+      labels:
+        run: coraza-spoa
+    spec:
+      containers:
+      - name: coraza-spoa
+        image: YOU-MUST-PROVIDE-YOUR-OWN-IMAGE
```

Comment:
Just pushed `docker.io/jcmoraisjr/coraza-spoa:experimental` and also `quay.io/jcmoraisjr/coraza-spoa:experimental`, adding PR 36 (which I'll add some comments shortly) and the diff below:

```diff
diff --git a/internal/request.go b/internal/request.go
index 841e924..c1fb139 100644
--- a/internal/request.go
+++ b/internal/request.go
@@ -51,13 +51,13 @@ func (s *SPOA) processRequest(msg spoe.Message) ([]spoe.Action, error) {
 			app, ok = s.applications[arg.Value.(string)]
 			if !ok {
 				if len(s.defaultApplication) > 0 {
-					fmt.Printf("application %q not found, using default application %q\n", arg.Value.(string), s.defaultApplication)
 					app, ok = s.applications[s.defaultApplication]
 					if !ok {
-						return nil, fmt.Errorf("default application %q not found", s.defaultApplication)
+						return nil, fmt.Errorf("default application not found: %s", s.defaultApplication)
 					}
+					app.logger.Debug("application not found, using default", zap.Any("application", arg.Value), zap.String("default", s.defaultApplication))
 				} else {
-					return nil, fmt.Errorf("application %q not found", arg.Value.(string))
+					return nil, fmt.Errorf("application not found: %v", arg.Value)
 				}
 			}
 		case "id":
```

This should be temporary, lets make the official image happen.

Reply:
Forgot to comment in the respective lines - if changing the default image, need to remove (or change) the last paragraph from the doc as well.